pkg/helm/actions/template_test.go

```diff
@@ -0,0 +1,60 @@
+package actions
+
+import (
+	"encoding/json"
+	"github.com/stretchr/testify/assert"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chartutil"
+	kubefake "helm.sh/helm/v3/pkg/kube/fake"
+	"helm.sh/helm/v3/pkg/storage"
+	"helm.sh/helm/v3/pkg/storage/driver"
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+func TestRenderManifests(t *testing.T) {
+	chart := "../testdata/influxdb-3.0.2.tgz"
+	store := storage.Init(driver.NewMemory())
+
+	actionConfig := &action.Configuration{
+		Releases:     store,
+		KubeClient:   &kubefake.PrintingKubeClient{Out: ioutil.Discard},
+		Capabilities: chartutil.DefaultCapabilities,
+		Log:          func(format string, v ...interface{}) {},
+	}
+
+	act := NewHelmActions(actionConfig)
+	rel, err := act.RenderManifests("test", chart, nil)
+	if err != nil {
+		t.Error(err)
+	}
+
+	assert.True(t, strings.HasPrefix(rel, "---"))
```

Comment:
this assertion is very fragile and does not tell you much about the returned result, and could be green in many wrong cases (e.g. the simplest one being just a response containing `---`). What kind of response we should get. First, it must be  a valid yaml content, second, given the defined chart, it should correspond to it. Thus, please choose/craft a simple chart, and assert that returned content fully.

Reply:
removed this assertion